app.env.example

```diff
@@ -47,3 +47,10 @@ STORAGE_BUCKET=
 MESSAGING_SENDERID=
 APP_ID=
 MEASUREMENT_ID=
+
+# whitelist ip with comma separated
+ALLOWED_HOSTS=ip
+
+# whitelist telegram ip_block ranges with comma separated
+# https://ipinfo.io/AS62041#block-ranges
+ALLOWED_HOST_BLOCKS=149.154,91.108,95.161
```

Comment:
@hengsokly please consider uses the ipaddress gem here(https://www.rubydoc.info/gems/ipaddress/0.8.0/IPAddress/IPv4)